client/components/filters/filters.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
I don't think we need this filter. This can be doing using css without a hardcoded width.

https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/

in our case:

student.scss 
```
.student {
  display: flex;
  flex-direction: column;
  height: 100%;
  @media(min-width: 768px) {
    .info {
      height: 108px;
      display: flex;
      flex-direction: column;
      justify-content: space-around;
    }
  }
  @media(max-width: 768px) {
    .info {
      padding: 15px 0;
      min-height: 324px;
    }
  }
  .interventions,
  .outreaches {
    flex: 1 0 0;
    padding: 15px 0;
    .toggle {
      cursor: pointer;
    }
    .invisible-menu {
      background: none !important;
      border: none !important;
      box-shadow: none !important;
    }
    .archived {
      background-color: lightgrey;
    }
    .truncate {
      white-space: nowrap;
      overflow: hidden;
      text-overflow: ellipsis;
    }
  }
}
```

interventions.html
```
...
          <div class="input-group">
            <input type="text" class="form-control truncate" readonly value="{{note.note}}"/>
            <span class="input-group-addon">{{note.date | date:'yyyy-MM-dd'}}</span>
          </div>
...
```
outreaches.html
```
...
          <div class="input-group">
            <input type="text" class="form-control truncate" readonly value="{{note.note}}"/>
            <span class="input-group-addon">{{note.date | date:'yyyy-MM-dd'}}</span>
            <span class="input-group-btn">
              <button class="btn btn-info" ng-click="viewNote(note, student)">
                <i class="fa fa-search"></i>
              </button>
            </span>
          </div>
...
```